test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1310,6 +1363,64 @@ def check_if_sz_vav(model, zone)
     return zone_system_check
   end
 
+  # Check if preheat coil control for system 5 through 8 are implemented
+  #
+  # @param baseline_base [Hash] Baseline
+  def check_preheat_coil_ctrl(baseline_base)
+    baseline_base.each do |baseline, model_baseline|
+      building_type, template, climate_zone, mod = baseline
+
+      # Concatenate modifier functions and arguments
+      mod_str = mod.flatten.join('_') unless mod.empty?
+
+      htg_coil_node_list = []
+      model_baseline.getAirLoopHVACs.each do |airloop|
+        # Baseline system type identified based on airloop HVAC name
+        if airloop.name.to_s.include?('Sys5') ||
+            airloop.name.to_s.include?('Sys6') ||
+            airloop.name.to_s.include?('Sys7') ||
+            airloop.name.to_s.include?('Sys8')
```

Comment:
Nothing wrong with that block of code but we had decided to move away from using the name to identify the system type and use additional properties (available at the thermal zone and air loop level) to get it. Here's an example: `system_type = air_loop_hvac.additionalProperties.getFeatureAsString('baseline_system_type').get`

Reply:
Updated